components/tabs/tab.js

```diff
@@ -0,0 +1,21 @@
+import { html, LitElement } from 'lit';
+import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+import { TabMixin } from './tab-mixin.js';
+
+class Tab extends TabMixin(SkeletonMixin(RtlMixin(LitElement))) {
+
+	render() {
```

Comment:
The way the mixin is defined, I think we'd just want this in this wip component:

```javascript
renderContent() {
	return html`
		<div class="tab-handler">
			<slot></slot>
		</div>
	`;
}
```

Otherwise, this `render` method overrides the one in the mixin and we don't get any of the base rendering.